components/UserList.js

```diff
@@ -42,36 +42,36 @@ function UserList ({ summaryView, type }) {
       return
     }
     // TODO: Can we replace with useSWR (might only be possible for fetching/GET)?
-    const result = await secureFetch(
-      `${_getUrl(type)}/${user.id}`,
+    const deleteResult = await secureFetch(
+      `${getUserUrl(type)}/${user.id}`,
       accessToken,
       'delete'
     )
-    mutate(_getUrl(type))
-    if (result.code >= 400) {
-      window.alert(result.message)
+    mutateList()
+    if (deleteResult.code >= 400) {
+      window.alert(deleteResult.message)
     }
   }
-  const onCreateAdminUser = async () => {
+  const onUpdateUser = async (args) => {
```

Comment:
I don't really want to destructure this if that's what you're looking for (just because we don't have static typing on this library and if the args change it could cause headaches). I'm not sure there's a much better name for this... `updateUserPayload`? I think `args` is OK.